### ISSUE TITLE
`include!`'d source, when encountering a parse error, does not report-and-abort.

### ISSUE THREAD

**Issue by cmr** [rust-lang/rust#21146]
Testcase:

`inc.rs`:
```rust
afddafadsfasdfdasfaf
fn never_parsed() -> i32 { 42 }
```

`main.rs`:
```rust
include!("inc.rs");
fn main() { never_parsed(); }
```

Output:

```
csguest@itl:~/Downloads$ rustc main.rs
main.rs:4:2: 4:14 error: unresolved name `never_parsed`
main.rs:4 	never_parsed();
          	^~~~~~~~~~~~
error: aborting due to previous error
```

Should output, instead, the parse errors from `include!`. Probably misplacing the diagnostic emitter somehow, when creating the nested parser for the included source, and doesn't have an `abort_if_errors`.

**Comment by cmr** [comment]
(I suspect this is an easy fix, but am not *quite* sure)
